ci/azure-jobs-linux.yml

```diff
@@ -30,7 +33,13 @@ jobs:
       # Pass modified PATH to following steps:
       echo "##vso[task.setvariable variable=PATH]$PATH"
     displayName: Install Dependencies
-  - bash: ./ci/build_librepcb.sh
+  - bash: rm -rf libs/quazip
+    displayName: Remove unbundled quazip
+    condition: and(succeeded(), contains(variables['UNBUNDLE'], 'quazip'))
```

Comment:
What's the reason for removing the *bundled* (not *unbundled*, as written) quazip?

Reply:
I remove the directory to ensure that the bundled library isn't used. The text is wrong though, it should say "bundled" :slightly_smiling_face: